### PR TITLE
[script.colorbox] 1.0.10

### PR DESCRIPTION
**Pull request by BADMS**
Hopefully within guidelines for 1 commit per pull.

### CODE REVIEW

**Comment by learningit** [comment]
This looks OK to me, but I'm not sure that all the assumptions that are made about cache files are valid in newer versions of Kodi.  Someone with more knowledge will need to comment on that.

**Comment by BADMS** [comment]
Well they were fine with first release for Jarvis [and have remained the same since].
Please direct me to documentation otherwise, thanks!

File: script.colorbox/Readme.md [hunk 1]
```diff
@@ -75,15 +81,29 @@ script.colorbox
...
+  
+  SetProperty(cfa_daemon_set,Pixelate,home)
+  
+  SetProperty(cfa_daemon_set,Posterize,home)
+  
+  SetProperty(cfa_daemon_set,Distort,home)
+  
+  This MUST be used as of now.
+  NONE will be processed if not set.
+  Next version will default to ALL being processed if not set!
```
Comment (phil65) [inline script.colorbox/Readme.md]: this doesnt sound good. Default = off should be preferred.

Comment (BADMS) [inline script.colorbox/Readme.md]: noted, will default off with and add property to call all. thanks

**Pull request merged**